app/containers/CreateCommunity/index.js

```diff
@@ -0,0 +1,157 @@
+/**
+ *
+ * CreateCommunity
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Helmet } from 'react-helmet';
+import { createStructuredSelector } from 'reselect';
+import { translationMessages } from 'i18n';
+import { compose } from 'redux';
+
+import injectSaga from 'utils/injectSaga';
+import injectReducer from 'utils/injectReducer';
+import { getBlob } from 'utils/profileManagement';
+
+import { makeSelectLocale } from 'containers/LanguageProvider/selectors';
+import Wrapper from 'containers/Profile/Wrapper';
+
+import * as selectors from './selectors';
+import reducer from './reducer';
+import saga from './saga';
+import messages from './messages';
+
+import {
+  uploadImageFileAction,
+  saveImageChanges,
+  clearImageChanges,
+  createCommunity,
+  setDafaultStore,
+} from './actions';
+
+import { NAME_FIELD, DESCRIPTION_FIELD } from './constants';
+
+import CreateCommunityForm from './CreateCommunityForm';
+
+/* eslint-disable react/prefer-stateless-function */
+export class CreateCommunity extends React.Component {
+  componentWillUnmount() {
+    this.props.setDafaultStoreDispatch();
+  }
+
+  createCommunity = (...args) => {
+    const { reset } = args[2];
+    const community = {
+      avatar: this.props.cachedImgHash,
+      name: args[0].get(NAME_FIELD),
+      description: args[0].get(DESCRIPTION_FIELD),
+    };
+
+    this.props.createCommunityDispatch(community, reset);
+  };
+
+  /* eslint no-empty: 0 */
+  uploadImage = event => {
```

Comment:
`uploadImage` and `getCroppedAvatar` is duplicated in CreateCommunity and EditProfile components. Try to create shared utility functions for reuse.